addons/api/addon/utils/indexed-db-query.js

```diff
@@ -223,6 +223,14 @@ const getKey = (key) => {
   }
 };
 
+const cleanValues = (values) => {
```

Comment:
Can you add a comment explaining why we need this?

Reply:
Yes. We were getting `null` values from the model (related to the peekRecord and findRecord issue). So the indexed-db-query was getting input that looked like this: `{ equals: null }`. So this `cleanValues` method will remove those `null` or `undefined` values if passed in.

Reply:
I don't understand, null is a valid value isn't it? Why are we cleaning it here?

Reply:
While manually testing we were getting a `null` value from `session.target_id`. Happy to revisit this after I address the other feedback.

Reply:
After discussion, we are removing this for now and going to address this in a later PR